cpp/src/datetime/datetime_ops.cu

```diff
@@ -180,6 +191,101 @@ std::unique_ptr<column> apply_datetime_op(column_view const& column,
   return output;
 }
 
+struct add_calendrical_months_functor {
+  column_view timestamp_column;
+  column_view months_column;
+  mutable_column_view output;
+
+  add_calendrical_months_functor(column_view tsc, column_view mc, mutable_column_view out)
+    : timestamp_column(tsc), months_column(mc), output(out)
+  {
+  }
+
+  // std chrono implementation is copied here due to nvcc bug
+  // https://howardhinnant.github.io/date_algorithms.html#days_from_civil
```

Comment:
license for this code included?

Reply:
i'm not sure what needs to be included here. a swipat has been filed and approved for this. are you suggesting that [this](https://github.com/HowardHinnant/date/blob/master/LICENSE.txt) license be included in this code?

Reply:
The SWIPAT approval should have included what the requirements for attribution are.

Reply:
it looks like the apache 2.0 license is sufficient for this case. i have updated the year in the license to 2020.